gatekeeper/manifest.json

```
@@ -19,7 +19,7 @@
     "configuration & deployment"
   ],
   "type": "check",
-  "is_public": false,
+  "is_public": true,
```

leave this as false for now to avoid migration issues
```suggestion
  "is_public": false,
```